src/containers/App/App.test.js

```diff
@@ -39,16 +41,69 @@ describe('App', () => {
   })
 
   it('should match the correct snapshot if there is an error', () => {
-    let mockProps2 = {
-      notes: [{ id: 1, title: 'Note Title' }],
-      items: [],
-      error: 'Something went wrong',
-      fetchNotes: jest.fn()
-    }
-    wrapper = shallow(<App {...mockProps2} />)
+    wrapper.setProps({error: 'Something went wrong'})
+    wrapper = shallow(<App {...mockProps} />)
     expect(wrapper).toMatchSnapshot()
   })
 
+  describe('Routes', () => {
+
+    it('should render the NoteArea container when at the root route', () => {
+      const wrapper = mount(
+        <Provider store={store}>
+          <MemoryRouter initialEntries={['/']}>
+            <App />
+          </MemoryRouter>
+        </Provider>
+      )
+      expect(wrapper.find(NoteArea)).toHaveLength(1)
```

Comment:
that's slick, I never would have thought of that

Reply:
Yeah I have also not seen it tested this way - will definitely be sure to reference this on our next project!